wegodang/products/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import jwt
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from products.models  import Category, Product, ProductModel, ProductImage
```

Comment:
사용하지 않고 있는 class는 제거하셔도 될 것 같습니다.

```suggestion
from products.models  import Category, Product
```

Reply:
수정하였습니다 !